src/components/PageLayout/Navbar/NavbarMenu/NavbarCollapseContent.vue

```diff
@@ -35,11 +35,8 @@
       [text: string]: {[text: string]: string} | string;
     };
 
-    loggedIn = false;
-
-    @Watch('$route')
-    checkLoginStatus() {
-      this.loggedIn = this.$store.state.userStore.loggedIn;
+    get loggedIn() {
+      return this.$vxm.user.loggedIn;
```

Comment:
The reason I waited for a route change is that before SSR, `user.loggedIn` would have been set to `true` before the new page loaded, and I wanted everything to change at once.
Did that change on the SSR version?

Reply:
The issue with that is then it won't get updated on the initial `authenticate` call, so if you're already logged in, it won't reflect that until you change pages

Reply:
How about `@Watch('$route', { immediate: true })`?

Reply:
I want the page to only load after the authentication is done, so that the login menu doesn't disappear after it was loaded; I'll have to check whether that's how it is now, but it wouldn't make sense otherwise in my opinion.

Reply:
That will still get fired off before the authenticate request has completed won't it?